data/org.freedesktop.portal.Background.xml

```diff
@@ -103,6 +103,29 @@
       <arg type="o" name="handle" direction="out"/>
     </method>
 
+    <!--
+      SetStatus:
+      @options: Vardict with optional further information
+
+      Sets the status of the application running in background.
+
+      Supported keys in the @options vardict include:
+      <variablelist>
+        <varlistentry>
+          <term>message s</term>
+          <listitem><para>
+            A string that will be used as the status message of the
+            application. Must be shorter than 96 characters.
```

Comment:
Ah, you have some of that here

Reply:
Yes, but I've applied your previous suggestion, and made the validation a bit more strict on the portal too